platformio.ini

```diff
@@ -45,6 +45,28 @@ upload_protocol = espota
 upload_port = owie.local
 ;board_build.gzip_fw = true
 
+[env:nodemcuv2]
```

Comment:
No need to copy all the details, you can inherit from the wemos env and just change the board. Looks at the `ota` env as an example.

Reply:
I'm going to be doin a lot of work today so I'll get it cleaned up on the next commit.